onnxruntime/python/tools/quantization/tensor_quant_overrides.py

```diff
@@ -0,0 +1,214 @@
+# -------------------------------------------------------------------------
```

Comment:
Note: moved tensor quant overrides helpers from base_quantizer.py to this new file.